packages/utils/src/instrument/fetch.ts

```diff
@@ -95,6 +111,55 @@ function instrumentFetch(): void {
   });
 }
 
+function instrumentFetchBodyReceived(): void {
+  if (!supportsNativeFetch()) {
+    return;
+  }
+
+  fill(GLOBAL_OBJ, 'fetch', function (originalFetch: () => void): () => void {
+    return function (...args: any[]): void {
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+      return originalFetch.apply(GLOBAL_OBJ, args).then(async (response: Response) => {
+        // clone response for awaiting stream
+        let clonedResponseForResolving: Response | undefined;
+        try {
+          clonedResponseForResolving = response.clone();
+        } catch (e) {
+          // noop
+          DEBUG_BUILD && logger.warn('Failed to clone response body.');
+        }
+
+        if (clonedResponseForResolving && clonedResponseForResolving.body) {
+          const responseReader = clonedResponseForResolving.body.getReader();
```

Comment:
Can we find a way to abort this after some time/when not needed anymore? E.g. using `controller.close()` or something like this? https://developer.mozilla.org/en-US/docs/Web/API/Streams_API/Using_readable_streams#reading_the_stream

Reply:
I can call the `read()` method in a `Promise.race`, what would be a reasonable timeout value? 5s?

Reply:
Hmm good question, I think we can do something longer too, e.g. 15s or something like this, but no strong feelings! Worst case if it is aborted "too early" it will just not update so not too bad :)